### PR TITLE
add support for selecting rows

### PR DESCRIPTION
**Pull request by wavemechanics**
### What

Add support for rows= query arguments to select specific rows to be returned.
rows=col:val1[,val2 ...]
rows=col:val1...val2

Multiple rows= arguments can be passed. All are OR'd in the resulting SQL WHERE clause.

### How to review

Check code for sanity, test the lambda (I've deployed it.)

### Who can review

Probably Steve or Viv are the only ones who care.

### CODE REVIEW

File: pkg/where/where.go [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+)
+
+// By default all rows are returned.
+// This can be limited by providing one or more rows= query variables.
+//
+// Each rows variable looks like this:
+//
+//	rows=col:rowspec[,rowspec]
+//
+// Where col is a column name, and rowspec is one of:
```
Comment (VivianAllen) [inline pkg/where/where.go:18]: can rowspec be an array? Looks like it, but the comment makes it sound like it could only be a single value OR a range?

Comment (wavemechanics) [reply pkg/where/where.go:18]: As it stands, rowspec is either a single geographic id or two ids separated by "...". Multiple rowspecs are separated by commas.

Comment (VivianAllen) [reply pkg/where/where.go:18]: so would `rows=geography_code:e1,e2,e3` be a valid query? (if those were valid geography codes)

**Review by VivianAllen — APPROVED** [review]
code looks good, tests look good, tests pass ✅

**Pull request merged**